figma-shop/product-json.ts

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
For now this is okay that you have such `json` in your project, but it has to be on firebase DB.

Reply:
Already deleted this file, it was only as an example